package.json

```diff
@@ -18,6 +18,8 @@
   "scripts": {
     "dev": "npm run build && qode dist/demo.js",
     "postinstall": "npm run build:addon",
+    "prepare": "npm run prettier && npm run build",
+    "prettier": "npx prettier --write --parser typescript --single-quote false --tab-width 2 --print-width 120 \"src/**/*.ts\"",
```

Comment:
Move this to prettier config instead.
Please remove prepare script. I want to keep the npm scripts area as lean and clean as possible.